open-company/open-company-web
# Follow-ups

Related PRs:
- [Storage](https://github.com/open-company/open-company-storage/pull/211)
- [Digest](https://github.com/open-company/open-company-digest/pull/25)
- [Email](https://github.com/open-company/open-company-email/pull/54)
- [Bot](https://github.com/open-company/open-company-bot/pull/87)
- [Lib](https://github.com/open-company/open-company-lib/pull/68)
- [Notify](https://github.com/open-company/open-company-notify/pull/12)

To test:
- add a post with follow-ups in a team section
- [x] default follow-up are all users but you
- [x] make sure all other users get an in app notification
- [x] make sure all other users get an email or a Slack notification
- add a post with follow-ups in a private section
- [x] default follow-up are all section users but you
- [x] make sure all other users get an in app notification
- [x] make sure all other users get an email or a Slack notification
- publish a post with only some follow-ups (not all possible users)
- edit the post and include the remaining follow-ups
- [x] make sure the added users get the notification
- complete some follow-ups of a post
- edit the post
- [x] make sure you can't remove the follow-ups for the user that have completed them
- [x] make sure you can't remove follow-ups for users that have added it for themself
